user1, 2019-08-14 10:53:35.032254
user1, 2019-08-14 10:55:53.559885
user1, 2019-08-14 10:57:47.230193
user2, 2019-08-16 22:53:08.750204
user2, 2019-08-16 22:54:54.245375
Bill, 2019-08-16 22:56:29.304352
Bill, 2019-08-16 22:57:12.765061
user3, 2019-08-16 22:58:40.589809
user2, 2019-08-16 23:01:54.355371
Bill, 2019-08-16 23:02:19.041352
Bill, 2019-08-16 23:03:25.325061
user3, 2019-08-16 23:05:37.889409
luvo, 2019-08-27 16:13:00.365508
user1, 2019-08-27 18:58:36.835177
user3, 2019-09-03 22:43:03.089699
luvo, 2019-09-04 10:37:19.052698
Internal Server Error: /product/91/
Traceback (most recent call last):
  File "C:\Users\User\djangogirls\myvenv\lib\site-packages\django\core\handlers\exception.py", line 35, in inner
    response = get_response(request)
  File "C:\Users\User\djangogirls\myvenv\lib\site-packages\django\core\handlers\base.py", line 128, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "C:\Users\User\djangogirls\myvenv\lib\site-packages\django\core\handlers\base.py", line 126, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\Users\User\djangogirls\ecommerce\views.py", line 64, in ProductPage
    attr_id = int(request.session['item'])
  File "C:\Users\User\djangogirls\myvenv\lib\site-packages\django\contrib\sessions\backends\base.py", line 55, in __getitem__
    return self._session[key]
KeyError: 'item'
luvo, 2019-09-04 21:38:00.170382
luvo, 2019-09-04 21:38:02.055046
Mpumi, 2019-09-05 16:22:01.710157
Forbidden (CSRF token missing or incorrect.): /sign-up/
luvo, 2019-09-05 19:33:26.779973
Internal Server Error: /my-shop/update-product/product/46/
Traceback (most recent call last):
  File "C:\Users\User\djangogirls\myvenv\lib\site-packages\django\core\handlers\exception.py", line 35, in inner
    response = get_response(request)
  File "C:\Users\User\djangogirls\myvenv\lib\site-packages\django\core\handlers\base.py", line 128, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "C:\Users\User\djangogirls\myvenv\lib\site-packages\django\core\handlers\base.py", line 126, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\Users\User\djangogirls\myvenv\lib\site-packages\django\contrib\auth\decorators.py", line 21, in _wrapped_view
    return view_func(request, *args, **kwargs)
  File "C:\Users\User\djangogirls\myvenv\lib\site-packages\django\contrib\auth\decorators.py", line 21, in _wrapped_view
    return view_func(request, *args, **kwargs)
  File "C:\Users\User\djangogirls\profiles\views.py", line 206, in UpdateProduct
    ProductInlineformset = inlineformset_factory(Product, ProductImage)
  File "C:\Users\User\djangogirls\myvenv\lib\site-packages\django\forms\models.py", line 1080, in inlineformset_factory
    FormSet = modelformset_factory(model, **kwargs)
  File "C:\Users\User\djangogirls\myvenv\lib\site-packages\django\forms\models.py", line 864, in modelformset_factory
    "Calling modelformset_factory without defining 'fields' or "
django.core.exceptions.ImproperlyConfigured: Calling modelformset_factory without defining 'fields' or 'exclude' explicitly is prohibited.
luvo, 2019-09-05 21:38:07.031577
luvo, 2019-09-06 18:28:23.332750
user3, 2019-09-06 18:32:25.080437
user3, 2019-09-06 18:58:04.765668
luvo, 2019-09-06 19:00:17.866975
luvo, 2019-09-10 15:39:11.057692

luvo, 2019-09-14 13:08:13.378010
MpumiD, 2019-09-14 19:07:26.042899
Internal Server Error: /register-shop/
Traceback (most recent call last):
  File "/home/luvo/djangogirls/venv/lib/python3.6/site-packages/django/core/handlers/exception.py", line 34, in inner
    response = get_response(request)
  File "/home/luvo/djangogirls/venv/lib/python3.6/site-packages/django/core/handlers/base.py", line 115, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/home/luvo/djangogirls/venv/lib/python3.6/site-packages/django/core/handlers/base.py", line 113, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/luvo/djangogirls/venv/lib/python3.6/site-packages/django/contrib/auth/decorators.py", line 21, in _wrapped_view
    return view_func(request, *args, **kwargs)
  File "/home/luvo/djangogirls/profiles/views.py", line 137, in RegisterShop
    return render(request, 'profiles/register-shop.html', context)
  File "/home/luvo/djangogirls/venv/lib/python3.6/site-packages/django/shortcuts.py", line 36, in render
    content = loader.render_to_string(template_name, context, request, using=using)
  File "/home/luvo/djangogirls/venv/lib/python3.6/site-packages/django/template/loader.py", line 62, in render_to_string
    return template.render(context, request)
  File "/home/luvo/djangogirls/venv/lib/python3.6/site-packages/django/template/backends/django.py", line 61, in render
    return self.template.render(context)
  File "/home/luvo/djangogirls/venv/lib/python3.6/site-packages/django/template/base.py", line 171, in render
    return self._render(context)
  File "/home/luvo/djangogirls/venv/lib/python3.6/site-packages/django/template/base.py", line 163, in _render
    return self.nodelist.render(context)
  File "/home/luvo/djangogirls/venv/lib/python3.6/site-packages/django/template/base.py", line 937, in render
    bit = node.render_annotated(context)
  File "/home/luvo/djangogirls/venv/lib/python3.6/site-packages/django/template/base.py", line 904, in render_annotated
    return self.render(context)
  File "/home/luvo/djangogirls/venv/lib/python3.6/site-packages/django/template/loader_tags.py", line 150, in render
    return compiled_parent._render(context)
  File "/home/luvo/djangogirls/venv/lib/python3.6/site-packages/django/template/base.py", line 163, in _render
    return self.nodelist.render(context)
  File "/home/luvo/djangogirls/venv/lib/python3.6/site-packages/django/template/base.py", line 937, in render
    bit = node.render_annotated(context)
  File "/home/luvo/djangogirls/venv/lib/python3.6/site-packages/django/template/base.py", line 904, in render_annotated
    return self.render(context)
  File "/home/luvo/djangogirls/venv/lib/python3.6/site-packages/django/template/loader_tags.py", line 62, in render
    result = block.nodelist.render(context)
  File "/home/luvo/djangogirls/venv/lib/python3.6/site-packages/django/template/base.py", line 937, in render
    bit = node.render_annotated(context)
  File "/home/luvo/djangogirls/venv/lib/python3.6/site-packages/django/template/base.py", line 904, in render_annotated
    return self.render(context)
  File "/home/luvo/djangogirls/venv/lib/python3.6/site-packages/django/template/base.py", line 987, in render
    output = self.filter_expression.resolve(context)
  File "/home/luvo/djangogirls/venv/lib/python3.6/site-packages/django/template/base.py", line 698, in resolve
    new_obj = func(obj, *arg_vals)
  File "/home/luvo/djangogirls/venv/lib/python3.6/site-packages/crispy_forms/templatetags/crispy_forms_filters.py", line 113, in as_crispy_field
    helper = getattr(field.form, 'helper', None)
AttributeError: 'str' object has no attribute 'form'
Internal Server Error: /register-shop/
Traceback (most recent call last):
  File "/home/luvo/djangogirls/venv/lib/python3.6/site-packages/django/core/handlers/exception.py", line 34, in inner
    response = get_response(request)
  File "/home/luvo/djangogirls/venv/lib/python3.6/site-packages/django/core/handlers/base.py", line 115, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/home/luvo/djangogirls/venv/lib/python3.6/site-packages/django/core/handlers/base.py", line 113, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/luvo/djangogirls/venv/lib/python3.6/site-packages/django/contrib/auth/decorators.py", line 21, in _wrapped_view
    return view_func(request, *args, **kwargs)
  File "/home/luvo/djangogirls/profiles/views.py", line 137, in RegisterShop
    return render(request, 'profiles/register-shop.html', context)
  File "/home/luvo/djangogirls/venv/lib/python3.6/site-packages/django/shortcuts.py", line 36, in render
    content = loader.render_to_string(template_name, context, request, using=using)
  File "/home/luvo/djangogirls/venv/lib/python3.6/site-packages/django/template/loader.py", line 62, in render_to_string
    return template.render(context, request)
  File "/home/luvo/djangogirls/venv/lib/python3.6/site-packages/django/template/backends/django.py", line 61, in render
    return self.template.render(context)
  File "/home/luvo/djangogirls/venv/lib/python3.6/site-packages/django/template/base.py", line 171, in render
    return self._render(context)
  File "/home/luvo/djangogirls/venv/lib/python3.6/site-packages/django/template/base.py", line 163, in _render
    return self.nodelist.render(context)
  File "/home/luvo/djangogirls/venv/lib/python3.6/site-packages/django/template/base.py", line 937, in render
    bit = node.render_annotated(context)
  File "/home/luvo/djangogirls/venv/lib/python3.6/site-packages/django/template/base.py", line 904, in render_annotated
    return self.render(context)
  File "/home/luvo/djangogirls/venv/lib/python3.6/site-packages/django/template/loader_tags.py", line 150, in render
    return compiled_parent._render(context)
  File "/home/luvo/djangogirls/venv/lib/python3.6/site-packages/django/template/base.py", line 163, in _render
    return self.nodelist.render(context)
  File "/home/luvo/djangogirls/venv/lib/python3.6/site-packages/django/template/base.py", line 937, in render
    bit = node.render_annotated(context)
  File "/home/luvo/djangogirls/venv/lib/python3.6/site-packages/django/template/base.py", line 904, in render_annotated
    return self.render(context)
  File "/home/luvo/djangogirls/venv/lib/python3.6/site-packages/django/template/loader_tags.py", line 62, in render
    result = block.nodelist.render(context)
  File "/home/luvo/djangogirls/venv/lib/python3.6/site-packages/django/template/base.py", line 937, in render
    bit = node.render_annotated(context)
  File "/home/luvo/djangogirls/venv/lib/python3.6/site-packages/django/template/base.py", line 904, in render_annotated
    return self.render(context)
  File "/home/luvo/djangogirls/venv/lib/python3.6/site-packages/django/template/base.py", line 987, in render
    output = self.filter_expression.resolve(context)
  File "/home/luvo/djangogirls/venv/lib/python3.6/site-packages/django/template/base.py", line 698, in resolve
    new_obj = func(obj, *arg_vals)
  File "/home/luvo/djangogirls/venv/lib/python3.6/site-packages/crispy_forms/templatetags/crispy_forms_filters.py", line 113, in as_crispy_field
    helper = getattr(field.form, 'helper', None)
AttributeError: 'str' object has no attribute 'form'
GET https://b5fc48c72aeb47f5b430c06c02ed26ed.eu-west-2.aws.cloud.es.io:9243/products/_search [status:200 request:0.736s]
GET https://b5fc48c72aeb47f5b430c06c02ed26ed.eu-west-2.aws.cloud.es.io:9243/categories/_search [status:200 request:0.081s]
GET https://b5fc48c72aeb47f5b430c06c02ed26ed.eu-west-2.aws.cloud.es.io:9243/shops/_search [status:200 request:0.084s]
luvo, 2019-09-14 19:27:13.316398
luvo, 2019-09-15 14:50:44.298747
luvo, 2019-09-15 15:53:03.361578
luvo, 2019-09-15 19:22:26.305340
Not Found: /my-cart/checkout/
Not Found: /product/5/undefined
luvo, 2019-09-27 17:40:13.279251

